README.md

```diff
@@ -41,9 +41,12 @@ Software tools or libraries, sorted by programming language.
   - [playground](https://shacl.org/playground/) - Browser-based testbed.
 - [rdf-validate-shacl](https://github.com/zazuko/rdf-validate-shacl) <img alt="NPM Version" src="https://img.shields.io/npm/v/rdf-validate-shacl" align="top"> <img alt="GitHub Release Date" src="https://img.shields.io/github/release-date/zazuko/rdf-validate-shacl" align="top"> - Supports SHACL Core; pure JavaScript validator on top of the [RDFJS](https://rdf.js.org/) stack; `MIT` license; `JavaScript`.
   - [playground](https://zazuko.github.io/shacl-playground) - Browser-based testbed.
-- [shacl-engine](https://github.com/rdf-ext/shacl-engine) <img alt="NPM Version" src="https://img.shields.io/npm/v/shacl-engine" align="top"> <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/rdf-ext/shacl-engine" align="top"> - Supports SHACL Core, SHACL-SPARQL; A fast engine for data provided as [RDF/JS](http://rdf.js.org/data-model-spec/) objects; `MIT` license; `JavaScript`.
+- [shacl-engine](https://github.com/rdf-ext/shacl-engine) <img alt="NPM Version" src="https://img.shields.io/npm/v/shacl-engine" align="top"> <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/rdf-ext/shacl-engine" align="top"> - Supports SHACL Core, SHACL-SPARQL; A fast engine for data provided as [RDF/JS](http://rdf.js.org/data-model-spec/) objects. 15-26x faster than other JavaScript or Python packages ([blog post](https://www.bergnet.org/2023/03/2023/shacl-engine/), [real-world examples](https://github.com/rdf-ext/shacl-engine/issues/12#issuecomment-1940875628)); `MIT` license; `JavaScript`.
   - [playground](https://playground.rdf-ext.org/shacl/) - Browser-based testbed.
   - [rdf-ext-cli](https://github.com/rdf-ext/rdf-ext-cli) - Command line tool for validation. Data and shapes can be given as file, URL, or SPARQL endpoint + query.
+  - Experimental SHACL 1.2 support
```

Comment:
Please shorten like this:

- Experimental SHACL 1.2 support: [shacl-engine experimental](https://github.com/rdf-ext/shacl-engine/tree/experimental), [playground experimental](https://playground.rdf-ext.org/shacl-experimental/)